nrf-hal-common/src/uarte.rs

```diff
@@ -109,7 +181,6 @@ where
 
         // Reset the events.
         self.0.events_endtx.reset();
-        self.0.events_txstopped.reset();
```

Comment:
Oops. Actually this was needed if we keep the blocking wait for txstopped below :sweat_smile: ... It should probably be moved next to that so it's more obvious why it's needed.

Reply:
🤣